pkgs/development/libraries/tiledb/default.nix

```diff
@@ -0,0 +1,32 @@
+{ pkgs ? import ./pkgs.nix }:
+
+with pkgs;
+
+  stdenv.mkDerivation rec {
+    name = "tiledb";
+    version = "1.5.1";
+
+    src = pkgs.fetchFromGitHub {
+      owner = "TileDB-Inc";
+      repo = "TileDB";
+      rev = "7236bb29f2aa1ac4de489f0791e9265cb257184d";
+      sha256 = "sha256:0ky0dcv1w1jn1cjn3819aq9xyd2wg80aagf2flxmd916flgr9zjl";
+  };
+
+  makeTarget = "tiledb";
+  outputs = ["py" "out"];
```

Comment:
Can you explain what the `py` target is for?

Reply:
during my investigation i found somewhere that we would need pyd library of tiledb to be imported by python wrapper. That is why i had added this.

Reply:
Can you point out to github where this py output is coming from?